ByLaws.md

```diff
@@ -46,19 +46,19 @@ Removal of Officers……………………………………………………
 Resignation of Officers………………………………………………….6
 Responsibilities of Officers…………………………………….6
 7.5.1            Chief Executive Officer………6
-         7.5.2            President………………….6
-         7.5.3            Chief Financial Officer / Treasurer……….………………….......6
-                 7.5.3(a) Books of Account…6
-                 7.5.3(b) Financial Reports…………………………………………………………………………..………………7
-                 7.5.3(c) Deposit and Disbursement of Money and Valuables….....7
+7.5.2                     President………………….6
+7.5.3 Chief Financial Officer / Treasurer…………………………6
+7.5.3(a)                   Books of Account…6
```

Comment:
this indentation is still not right. plus do it in the previous pull request by pushing to the same patch branch.